### PR TITLE
moved out tests and stories from example folder, restructured the project

### PR DESCRIPTION
**Pull request by thetaungg**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,10 +37,15 @@
     "run:example": "yarn workspace example start",
```
Comment (shashank1010) [inline package.json:37]: remove

File: package.json [hunk 2]
```diff
@@ -38,10 +37,15 @@
...
     "run:storybook": "yarn workspace example storybook",
```
Comment (shashank1010) [inline package.json:38]: replace

File: stories/components/Drawer/Drawer.stories.tsx [hunk 3]
```diff
@@ -440,61 +440,63 @@ DrawerItem.play = async ({ canvasElement }) => {
...
-    await expect(window.getComputedStyle(drawerCollapsibleItemHeader).backgroundColor).toBe(
-        defaultBg,
-    );
-    await expect(canvas.queryByTestId('drawer-collapsible-item-content')).toBeNull();
-    await expect(canvas.queryByTestId('Important')).toBeNull();
-    await expect(canvas.queryByTestId('All mails')).toBeNull();
-    await expect(canvas.queryByTestId('Spams')).toBeNull();
-    await expect(canvas.queryByTestId('Manage labels')).toBeNull();
-};
+// DrawerCollapsibleItem.play = async ({ canvasElement }) => {
```
Comment (shashank1010) [inline stories/components/Drawer/Drawer.stories.tsx:443]: Why have we disabled this?

**Review by shashank1010 — CHANGES_REQUESTED** [review]

File: stories/components/Drawer/Drawer.stories.tsx (hunk 3 above)
Comment (thetaungg) [reply stories/components/Drawer/Drawer.stories.tsx]: Good that you found it. It was by mistake

**Review by shashank1010 — APPROVED** [review]

**Pull request merged**